vue.config.js

```diff
@@ -15,8 +15,19 @@ const productionPlugins = [
 
 module.exports = {
   baseUrl: BASE_URL,
-  lintOnSave: false,
   configureWebpack: {
-    plugins: production ? productionPlugins : developmentPlugins
-  }
-}
\ No newline at end of file
+    plugins: production ? productionPlugins : developmentPlugins,
+  },
+  lintOnSave: false,
+  pwa: {
+    workboxOptions: {
+      runtimeCaching: [
+        {
+          handler: 'networkFirst',
+          urlPattern: new RegExp('.*'),
+        },
+      ],
+      skipWaiting: true,
```

Comment:
muy bueno! Yo creo que la linea de arriba no haria falta ahora. Si el sw se actualiza correctamente los cambios en los archivos cacheados se reflejan ni bien recargas la pagina.

Reply:
Habria que ver si algunos de los drawbacks que se explican aca https://redfin.engineering/how-to-fix-the-refresh-button-when-using-service-workers-a8e27af6df68 nos afectarian

Reply:
La línea de arriba es para que funcione cuando estas offline